.env.example

```diff
@@ -0,0 +1,16 @@
+# SERVER
+SERVER_PORT=3000
+
+# DOCKER
+WORKDIR=/usr/src/app/
+
+# STORAGE
+FILES_LOCATION=./data
+STORAGE=influx
+
+# INFLUX
+# Set to `aggr-influx` if you use docker
```

Comment:
does that mean aggr-influx = docker host ?
"Set to `aggr-influx` if you want to connect to the Influx instance within docker"